python/src/lib/workbook_validator.py

```diff
@@ -97,38 +103,45 @@ def validate(workbook: IO[bytes]) -> Tuple[Errors, Optional[str]]:
     See: https://openpyxl.readthedocs.io/en/stable/optimized.html
     If there is any trouble loading files from memory please see here: https://stackoverflow.com/questions/20635778/using-openpyxl-to-read-file-from-memory
     """
-    wb = load_workbook(filename=workbook, read_only=True)
+    try:
+        wb = load_workbook(filename=workbook, read_only=True)
+        return validate_workbook(wb)
```

Comment:
I've noticed sometimes failures that aren't captured are showing up as "processing". It would be great if we can add an except block like so:
```py
from src.lib.logging import get_logger

logger = get_logger()
try:
    wb...
    return validate_workbook(wb, logger)
except Error as e:
    logger.error(f'Unexpected Validation Error: {e}')
    return [WorkbookError(f'Unable to validate workbook. Please reach out to grants-helpdesk@usdigitalresponse.org')]
finally:
    workbook.close()
```

Reply:
Sounds good